client/src/extension.ts

```diff
@@ -47,9 +48,11 @@ export async function activate (context: vscode.ExtensionContext): Promise<void>
 
   loadLoggerSettings()
   bitbakeExtensionContext = context
-  bitbakeDriver.loadSettings(vscode.workspace.getConfiguration('bitbake'), vscode.workspace.workspaceFolders?.[0].uri.fsPath)
+  workspaceBitbakeConfig = vscode.workspace.getConfiguration('bitbake')
+  logger.debug('Loaded bitbake workspace settings: ' + JSON.stringify(workspaceBitbakeConfig))
+  bitbakeDriver.loadSettings(workspaceBitbakeConfig, vscode.workspace.workspaceFolders?.[0].uri.fsPath)
   bitBakeProjectScanner.setDriver(bitbakeDriver)
-  updatePythonPath(bitbakeDriver.bitbakeSettings.pathToBitbakeFolder)
+  updatePythonPath(workspaceBitbakeConfig.pathToBitbakeFolder)

```

Comment:
Following my above comment, I suggest removing the argument in `updatePythonPath` and run `workspaceBitbakeConfig = vscode.workspace.getConfiguration('bitbake')` directly there to avoid code duplication.
Also, since the reason for this change is not trivial, I'd suggest adding a comment on this assignment explaining why we do not use `bitbakeDriver.bitbakeSettings.pathToBitbakeFolder` directly anymore.